scripts/decredStakeVoting.md

```diff
@@ -1,46 +1,47 @@
+The 5 stake votes contained in each new block vote on the validity of the previous one. If this block is deemed invalid by at least a 3 to 5 majority, then its associated transactions and proof-of-work reward are stripped from the network while the proof-of-stake reward is retained. This process acts as a check on miner power, giving stakeholders a way to crack down on miners who are not following the rules and prevent miner aggressions such as block withholding or a 51% attack.
 
-In order to ensure that a vote is not missed, a stake server can be used.
-Stake servers allow stake voters to cast votes without worrying about missed votes due to computer downtime, bad internet connections, or inconsistent electricity grids.
+A ticket may also cast votes concerning other agenda items put up for a community vote. One example of an item that was approved in this manner was a consensus rule upgrade improving the algorithm which sets ticket price.
 
-Stake servers will vote on behalf of a stake voter for a small fee. This server fee helps pay for the cost of server infrastructure needed to ensure tickets vote quickly and reliably.
+In rare cases, it is possible for votes to be missed even though they were cast. This can happen when miners are processing blocks faster than the votes can propagate through the network. This leads to the mining of blocks with less than 5 votes. If a block is mined with less than the required 5 votes, the remaining votes destined for that block are considered ‘missed’ and the funds used to purchase them are unlocked.
 
-In rare cases, it is possible for votes to be missed even though they were cast. This can happen when miners are processing blocks faster than the votes can propagate through the network.
+After a stake voter's ticket has been drawn and their votes are successfully cast, the Decred spent on the ticket is unlocked and a reward for voting, called a ticket reward, is deposited into the ticket owners account. This ticket reward amounts to 6% of the total Decred block reward. After a waiting period of 20 hours, these funds become available for spending.
 
-Miners are incentivised to include the ideal 5 votes per block, but are only required to include the necessary minimum of 3 votes. A miner may only include 3 votes in a block *for some reason (can't seem to find explanation why this may happen)*. If this is the case and a block is mined with only 3 votes, the remaining 2 votes destined for that block are considered ‘missed’ and the funds used to purchase them are unlocked.
-To estimate the amount of Decred a stake voter can earn in a year through proof of stake mining, use this equation to calculate the annual percentage rate.
+Stake voting requires no action by the stakeholder other than purchasing tickets. By default, tickets are configured to `abstain` from voting on current agenda items, however, `yes` or `no` votes can be configured easily through the Decrediton staking wallet.
```

Comment:
move this detail up to so we can end with the exciting end results of all the upgrades that are possible